.github/workflows/IntegrationTest.yml

```diff
@@ -0,0 +1,52 @@
+name: IntegrationTest
```

Comment:
What's this file?

Reply:
It runs integration tests. Currently only with Distributions.

Reply:
The same action is eg used by ChainRulesCore, DiffRules, and many Turing and SciML packages.